src/common/resources/titleResources.js

```diff
@@ -9,3 +9,12 @@ export const titlesResource = {
   path: TITLES_API,
   accumulate: true,
 };
+
+export const identifierTypesResource = {
+  ...baseManifest,
+  path: 'identifier-types',
+  params: {
+    query: 'cql.allRecords=1 sortby name',
+  },
+  records: 'identifierTypes',
+};
```

Comment:
Do you need to add `okapiInterface` version to `package.json` for `identifierTypes`?

Reply:
right

Reply:
Don't forget about `CHANGELOG.md`